src/views/payment-sheet-views/apple-pay-view.js

```diff
@@ -40,7 +41,12 @@ ApplePayView.prototype.initialize = function () {
 
       global.ApplePaySession.canMakePaymentsWithActiveCard(self.applePayInstance.merchantIdentifier).then(function (canMakePayments) {
         if (!canMakePayments) {
-          self.model.reportError('applePayActiveCardError');
+          if (isProduction) {
+            self.model.reportError('applePayActiveCardError');
+          } else {
+            console.error('Could not find an active card. This may be because you're using a production iCloud account in a sandbox Apple Pay Session. Log in to a Sandbox iCloud account to test this flow, and add a card to your wallet. For additional assistance, visit  https://help.braintreepayments.com'); // eslint-disable-line no-console
```

Comment:
```suggestion
            console.error('Could not find an active card. This may be because you\'re using a production iCloud account in a sandbox Apple Pay Session. Log in to a Sandbox iCloud account to test this flow, and add a card to your wallet. For additional assistance, visit  https://help.braintreepayments.com'); // eslint-disable-line no-console
```

Reply:
ak, those darn escaped characters strike again. Thanks for that catch!